### ISSUE TITLE
Sticks both point to top right constantly.

### ISSUE THREAD

**Issue by FabersBruh** [Davidobot/BetterJoy#588]

**Comment by FreshStaticSn0w** [comment]
Can I ask what brand of controller this was? I'm having a similar issue in that cemu doesn't seem to be detecting my up and right inputs on either stick. When using the controller on my desktop (or in steam big picture) it constantly moves the cursor up and to the right (go figure). This error only occurs, however, when the controller is connected via bluetooth. When I plug the controller into my PC it seems to work just fine. Not sure what is going on here. The controller that I purchased was this one by a Chinese company called HIRIFULL : https://www.amazon.com/HIRIFULL-Wireless-Controller-Nintendo-Vibration/dp/B08BZP93K4

Any input from you or anyone else who comes across this would be much appreciated. Thanks!

**Comment by FabersBruh** [comment]
I honestly do not know.. I had gotten this from Aliexpress.. it practically has no brand and works quite fine with the switch.. :
https://www.aliexpress.com/item/4001272896723.html?spm=a2g0s.9042311.0.0.56144c4dK4s5R3

**Comment by FreshStaticSn0w** [comment]
Interesting. It's definitely something to do with the BetterJoyForCemu application, as the controller works perfectly fine (whether plugged in or using bluetooth) if the controller is **not connected to the app.** I'm going to go ahead and make a post myself and outline the issue that I/We're facing. I've seen David work well with folks who have issues with third party controllers in the past, so I'm hoping I can get in contact with him and that this is no different.

**Comment by FabersBruh** [comment]
May you manage it for both of us then. Hopefully it gets fixed and not require me to buy a dedicated controller for my pc .. heh

**Comment by Radnom44** [comment]
Hey guys, i have the same aliexpress controller as FabersBruh and have the same issue. hope they will fix it.

**Comment by shukenmg** [comment]
Try this:
 1. Download this [Build](https://github.com/Davidobot/BetterJoy/files/5434135/TestV2.zip) and set it up.
 2. Open the settings and change the `NonOriginalController` setting to "ControllerCalibration" without quotes then apply and test.

If that doesn't work please let us know.

**Comment by Radnom44** [comment]
i tried  but it doesn't work for me. @FabersBruh @FreshStaticSn0w ??

**Comment by Gerbil4444** [comment]
Same issue, but mine are generic third party left and right joycons:
https://www.amazon.com/Controller-Replacement-Redesigned-Ergonomic-Comfortable/dp/B083NKR7NF/ref=sr_1_18?dchild=1&keywords=joycon&qid=1604437328&sr=8-18&th=1

Trying the TestV2 build with the NonOriginalController setting changed to DefaultCalibration did fix the joystick issue, however it also sets both controllers as individual pro controllers.

**Comment by Radnom44** [comment]
you guys mean delete the " <add key="NonOriginalController" value="disabled" />" and replace with  <add key="ControllerCalibration" value="disabled" /> in the BetterJoyForCemu.exe config file  right?

**Comment by Radnom44** [comment]
you guys mean to delete the .....add key="NonOriginalController" value ="disabled".... and replace with ......add key="ControllerCalibratio" value="disabled"....... right??

**Comment by Radnom44** [comment]
you guys mean to delete the .....add key="NonOriginalController" value ="disabled".... and replace with ......add key="ControllerCalibration" value="disabled"....... right??

**Comment by Radnom44** [comment]
you guys mean to delete the .....add key="NonOriginalController" value ="disabled".... and replace with ......add key="DefaultCalibration" value="disabled"....... right??

**Comment by Radnom44** [comment]
you guys mean to delete the .....add key="NonOriginalController" value ="disabled".... and replace with ......add key="DefaultCalibration" value="disabled"....... right??

**Comment by Gerbil4444** [comment]
> you guys mean to delete the .....add key="NonOriginalController" value ="disabled".... and replace with ......add key="DefaultCalibration" value="disabled"....... right??

Change value to "DefaultCalibration" and leave the "NonOriginalController" key as is.

**Comment by Radnom44** [comment]
Ohh now its working thanks for the help!!!!

**Comment by FabersBruh** [comment]
I will be giving this a try tomorrow. Will let you know how it goes

**Comment by shukenmg** [comment]
> Same issue, but mine are generic third party left and right joycons:
> https://www.amazon.com/Controller-Replacement-Redesigned-Ergonomic-Comfortable/dp/B083NKR7NF/ref=sr_1_18?dchild=1&keywords=joycon&qid=1604437328&sr=8-18&th=1
> 
> Trying the TestV2 build with the NonOriginalController setting changed to DefaultCalibration did fix the joystick issue, however it also sets both controllers as individual pro controllers.

Yeah, it's how it is, for now, it might change in the future, so stay tuned :)

**Comment by FreshStaticSn0w** [comment]
AYYYY IT WORKED. 

All I had to do to get it to work was to change:
add key="NonOriginalController" value ="disabled"
to
add key="NonOriginalController" value ="DefaultCalibration"

Thank you all for being so helpful!!!

**Comment by maxill1** [comment]
I have a 3rd party pair of joycon and I was having the issue on Left Joycon Stick.

TestV2 version + key="NonOriginalController" value ="DefaultCalibration" solved the issue on Left JoyCon but introduced it on Right JoyCon Stick.

**Comment by maxill1** [comment]
> Same issue, but mine are generic third party left and right joycons:
> https://www.amazon.com/Controller-Replacement-Redesigned-Ergonomic-Comfortable/dp/B083NKR7NF/ref=sr_1_18?dchild=1&keywords=joycon&qid=1604437328&sr=8-18&th=1
> 
> Trying the TestV2 build with the NonOriginalController setting changed to DefaultCalibration did fix the joystick issue, however it also sets both controllers as individual pro controllers.

@Gerbil4444 can you the build posted in this pull request?

https://github.com/Davidobot/BetterJoy/pull/663#issuecomment-759652125

This _should_ allow you to change the controller type.
